src/main/java/jdbchomework/console/DevelopersMenu.java

```diff
@@ -0,0 +1,83 @@
+package jdbchomework.console;
+
+import jdbchomework.controller.DeveloperController;
+import jdbchomework.controller.SkillController;
+import jdbchomework.entity.Skill;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DevelopersMenu extends AbstractMenu {
+
+    private static final String PLEASE_ENTER_DEVELOPER_ID = "Please enter developer ID: ";
+    private static final String DEVELOPER_NOT_FOUND = "Developer not found";
+    private static final String PLEASE_ENTER_DEVELOPER_NAME = "Please enter developer name: ";
+
+    private DeveloperController developerController;
+    private SkillController skillController;
+
+    public DevelopersMenu(MenuContents menuContents, DeveloperController developerController,
+                          SkillController skillController) {
+        super(LoggerFactory.getLogger(DevelopersMenu.class), menuContents);
+        this.menuContents = menuContents;
+        this.developerController = developerController;
+        this.skillController = skillController;
+    }
+
+    public void menu() {
+        mark:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor the code to remove this label and the need for it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ALabelsShouldNotBeUsedCheck)